packages/terra-table/src/TableRowContent.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+const propTypes = {
+  /**
+   * Content to be displayed for the row cell
+   */
+  content: PropTypes.any.isRequired,
+  /**
+   * The maximum height for the cell content in a table
+   */
+  height: PropTypes.oneOf(['tiny', 'small', 'medium', 'large', 'huge']),
+};
+
+const TableRowContent = ({
+  content,
+  height,
+  ...customProps
+  }) => {
+  const contentClassName = classNames([
+    { [`terra-Table-row-max-height--${height}`]: height },
+    'terra-Table-cell',
+    customProps.className,
+  ]);
+
+  return (
+    <td {...customProps} className={contentClassName}>
+      <div>{content}</div>
```

Comment:
By default, the minimum height set on the table cell is the minimum height required by the content in the cell. Therefore in order to restrict the maximum height of the cell I have to add extra div tag.